consensus/safety-rules/tests/binary.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use libra_config::{
+    config::{NodeConfig, RemoteService, SafetyRulesService, PersistableConfig},
+    utils,
+};
+use libra_types::{validator_signer::ValidatorSigner};
+use safety_rules::{test_utils, SafetyRulesManager};
+use std::net::{IpAddr, Ipv4Addr, SocketAddr};
+
+const BINARY: &str = env!("CARGO_BIN_EXE_safety-rules");
+
+#[test]
+fn test_consensus_state() {
+    let mut config = NodeConfig::random().consensus.safety_rules;
+    let test_config = config.test.as_mut().unwrap();
+    let private_key = test_config.consensus_key.as_ref().unwrap().private_key();
+    let signer = ValidatorSigner::new(test_config.author, private_key);
+    let waypoint = test_utils::validator_signers_to_waypoint(&[&signer]);
```

Comment:
Is there no way to get this information without this particular test_util? If so then you could drop the requirement to build with the testing feature flag

Reply:
I mean we could generate these somehow else by copypastaing stuff around, but these values are required to start the binary and keep it running.

Reply:
I'm specifically talking about this single function here, `test_utils::validator_signers_to_waypoint`. But yeah i understand.